api/v1alpha1/k8s_types.go

```diff
@@ -76,7 +77,7 @@ type VolumeSpec struct {
 	// The PVC will be created based on the PersistentVolumeClaimSpec provided
 	// to the VolumeSpec.
 	VolumeSource corev1.VolumeSource `json:"volumeSource"`
-
+	VolumeDevice corev1.VolumeDevice `json:"volumeDevice"`
```

Comment:
Can you please add some description for this field?

Reply:
Sure